forc-tracing/src/telemetry.rs

```diff
@@ -0,0 +1,59 @@
+//! Telemetry utilities for logging to InfluxDB
+
+pub use fuel_telemetry::*;
```

Comment:
Unconditionally re-exporting `fuel_telemetry::*` forces all consumers to compile the dependency even when the telemetry feature is off;
```suggestion
#[cfg(feature = "telemetry")]
pub use fuel_telemetry::*;
```

Reply:
Oh 🤦  catch thanks!